vtysh/vtysh.c

```diff
@@ -56,6 +56,9 @@ struct vty *vty;
 /* VTY shell pager name. */
 char *vtysh_pager_name = NULL;
 
+/* VTY should add timestamp */
+int vtysh_add_timestamp;
```

Comment:
bool and use vtysh_add_timestamp = true; or = false in the code instead of 1 and 0

Reply:
Ok will do. I put them together b/c I saw them as a single enhancement which was "parallelize the support bundle collection", the commong_config.py runs the .py processes (per router) in parallel, the generate_support_bundle.py runs the vtysh's in parallel -- the glue was that the output is a support bundle. The enhancements also stand on their own though so --- ok. :)